### PR TITLE
[DOC] replace skipped doctests with default code-blocks

### LINKED ISSUE #3681: [DOC] Use code blocks instead of doctests for rst files in `doc`
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe your proposed suggestion in detail.

I noticed a bunch of doctests marked for skipping in some rsts. Why not just use code blocks? I'm guessing they are not run to avoid making the build more expensive but it's not clear why they should be written as doctests. It seems there is a mixture of the two formats in the same file anyways

### List any pages that would be impacted.

```bash
❯ grep -rl doc -e 'doctest: +SKIP'
doc/plotting/index.rst
doc/connectivity/functional_connectomes.rst
doc/connectivity/connectome_extraction.rst
doc/decoding/decoding_intro.rst
doc/manipulating_images/input_output.rst
doc/building_blocks/neurovault.rst
doc/building_blocks/manual_pipeline.rst
```

### PR DESCRIPTION
**Pull request by psadil**
Closes #3681.

Changes proposed in this pull request:

- doctest blocks that had skipped lines were converted to (default) code-blocks
- output that could have been tested against via a doctest ([e.g.](https://github.com/psadil/nilearn/blob/6fb7df5c7af3ec9bcc447e2b3ee274df7273e370/doc/manipulating_images/input_output.rst?plain=1#L121-L127)) were converted into comments

### CODE REVIEW

File: doc/building_blocks/neurovault.rst [hunk 1]
```diff
@@ -82,30 +88,37 @@ Extra keyword arguments are treated as additional image filters, so if we want
...
-    >>> from nilearn.datasets import neurovault
-    >>> fmri = fetch_neurovault( # doctest: +SKIP
-    ... modality=neurovault.IsIn('fMRI-BOLD', 'fMRI-CBF', 'fMRI-CBV'), # doctest: +SKIP
-    ... max_images=100) # doctest: +SKIP
+.. code-block:: default
+
+  from nilearn.datasets import neurovault
+  fmri = fetch_neurovault(
+    ... modality=neurovault.IsIn('fMRI-BOLD', 'fMRI-CBF', 'fMRI-CBV'),
+    ... max_images=100)
```
Comment (Remi-Gau) [inline doc/building_blocks/neurovault.rst:105]: I don't think we need to keep the ellipsis in those code blocks

File: doc/building_blocks/neurovault.rst [hunk 2]
```diff
@@ -82,30 +88,37 @@ Extra keyword arguments are treated as additional image filters, so if we want
...
 We could also have used ``Contains`` :
 
-    >>> fmri = fetch_neurovault( # doctest: +SKIP
-    ... modality=neurovault.Contains('fMRI'), # doctest: +SKIP
-    ... max_images=7) # doctest: +SKIP
+.. code-block:: default
+
+  fmri = fetch_neurovault(
+    ... modality=neurovault.Contains('fMRI'),
+    ... max_images=7)
```
Comment (Remi-Gau) [inline doc/building_blocks/neurovault.rst:113]: I don't think we need to keep the ellipsis in those code blocks

File: doc/building_blocks/neurovault.rst [hunk 3]
```diff
@@ -82,30 +88,37 @@ Extra keyword arguments are treated as additional image filters, so if we want
...
 If we need regular expressions, we can also use ``Pattern`` :
 
-    >>> fmri = fetch_neurovault( # doctest: +SKIP
-    ... modality=neurovault.Pattern('fmri(-.*)?', neurovault.re.IGNORECASE), # doctest: +SKIP
-    ... max_images=7) # doctest: +SKIP
+.. code-block:: default
+
+  fmri = fetch_neurovault(
+    ... modality=neurovault.Pattern('fmri(-.*)?', neurovault.re.IGNORECASE),
+    ... max_images=7)
```
Comment (Remi-Gau) [inline doc/building_blocks/neurovault.rst:121]: I don't think we need to keep the ellipsis in those code blocks

File: doc/building_blocks/neurovault.rst [hunk 4]
```diff
@@ -127,9 +140,11 @@ job for images. The default values for these parameters don't filter out
...
 Using a filter rather than a dictionary, the first example becomes:
 
-    >>> bold = fetch_neurovault( # doctest: +SKIP
-    ...     image_filter=lambda meta: meta.get('modality') == 'fMRI-BOLD', # doctest: +SKIP
-    ...     image_terms={}, max_images=7) # doctest: +SKIP
+.. code-block:: default
+
+  bold = fetch_neurovault(
+    ... image_filter=lambda meta: meta.get('modality') == 'fMRI-BOLD',
+    ... image_terms={}, max_images=7)
```
Comment (Remi-Gau) [inline doc/building_blocks/neurovault.rst:147]: I don't think we need to keep the ellipsis in those code blocks

File: doc/building_blocks/neurovault.rst [hunk 5]
```diff
@@ -147,8 +162,10 @@ many metadata fields - say, an image should only be kept if its metadata has
 less than 50 fields.  This cannot be done by simply comparing each key in a
 metadata dictionary to a required value, so we need to write our own filter:
 
-  >>> small_meta_images = fetch_neurovault(image_filter=lambda meta: len(meta) < 50, # doctest: +SKIP
-  ...                                      max_images=7) # doctest: +SKIP
+.. code-block:: default
+
+  small_meta_images = fetch_neurovault(image_filter=lambda meta: len(meta) < 50,
+  ...                                      max_images=7)
```
Comment (Remi-Gau) [inline doc/building_blocks/neurovault.rst:168]: I don't think we need to keep the ellipsis in those code blocks

File: doc/plotting/index.rst [hunk 6]
```diff
@@ -604,24 +626,26 @@ use :func:`view_markers`.
...
+.. code-block:: default
 
-    >>> from nilearn import plotting, datasets     # doctest: +SKIP
-    >>> img = datasets.fetch_localizer_button_task()['tmap']     # doctest: +SKIP
-    >>> html_view = plotting.view_img(img, threshold=2, vmax=4, cut_coords=[-42, -16, 52],
-    ...                                     title="Motor contrast")     # doctest: +SKIP
+     from nilearn import plotting, datasets
+     img = datasets.fetch_localizer_button_task()['tmap']
+     html_view = plotting.view_img(img, threshold=2, vmax=4, cut_coords=[-42, -16, 52],
+     ...                                     title="Motor contrast")
```
Comment (Remi-Gau) [inline doc/plotting/index.rst:636]: I don't think we need to keep the ellipsis in those code blocks

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
I think we should remove the ellipsis (`...`) in the code blocks.

Do you mind fixing this?

**Comment by psadil** [comment]
> I think we should remove the ellipsis (`...`) in the code blocks.
> 
> Do you mind fixing this?

Will do!

**Comment by Remi-Gau** [comment]
@psadil Do you want to  update the changelog and add yourself to the list of contributors?

https://nilearn.github.io/dev/development.html#changelog

**Comment by psadil** [comment]
> @psadil Do you want to update the changelog and add yourself to the list of contributors?
> 
> https://nilearn.github.io/dev/development.html#changelog

Sure thing. In this pull request or another?

**Comment by Remi-Gau** [comment]
> Sure thing. In this pull request or another?

do it directly here: helps keep track of what changelog entry corresponds to what PR

**Review by Remi-Gau — APPROVED** [review]
Looks good to me.

**Review by ymzayek — APPROVED** [review]
Thx @psadil !

**Pull request merged**